src/selectors/per-thread/thread.js

```diff
@@ -310,10 +310,10 @@ export function getThreadSelectorsPerThread(
    * base thread, if your thread is the preview filtered thread.
    */
   const getSampleIndexOffsetFromPreviewRange: Selector<number> = createSelector(
```

Comment:
This selector is used when computing the timings in the sidebar: indeed we use the full thread in addition to the filtered thread in this function, and therefore we need this "offset" to be able to relate the samples in these 2 thread objects.

Reply:
Praise: Nice find!

Thought: Should we assert this behavior is correct in tests? I'll look for it in upcoming commits.

Reply:
Ah, I don't think I have tests for the sidebar and a profile with allocations. I'll try to add one.